### PR TITLE
Clcheles MRTK3 Tutorial

### PR DESCRIPTION
**Pull request by cheles**
Upgrade to 2021.3.28f1 / link fixes / TutorialCallbacks  methods to ignoreCase

### CODE REVIEW

**Review by AMollis — CHANGES_REQUESTED** [review]

File: Packages/manifest.json [hunk 1]
```diff
@@ -13,18 +13,18 @@
     "com.microsoft.mrtk.uxcomponents.noncanvas": "file:MixedReality/com.microsoft.mrtk.uxcomponents.noncanvas-3.0.0-pre.11.tgz",
     "com.microsoft.mrtk.uxcore": "file:MixedReality/com.microsoft.mrtk.uxcore-3.0.0-pre.11.tgz",
```
[the remark this answers is not in the document]
Comment (cheles) [reply Packages/manifest.json:14]: I kept this file more for Unity 2021.3.11f1 to 28f1 upgrade, but didn't tested with latest MRTK3 prev17. I can test tomorrow and repush if needed.

File: Assets/Tutorials/BoundsControl/BoundsControl-LastPage.asset [hunk 2]
```diff
@@ -41,7 +41,7 @@ MonoBehaviour:
           to a GameObject using the <b>BoundsControl</b> script. You can further
           modify the properties to customize the look and behavior of the bounds.
           Manipulation events can be added as well. \n\nBe sure to review our <a
-          href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-spatialmanipulation/packages/spatialmanipulation/bounds-control\">Bounds
+          href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-spatialmanipulation/packages/spatialmanipulation/bounds-control\">Bounds
```
Comment (aprilspeight) [inline Assets/Tutorials/BoundsControl/BoundsControl-LastPage.asset:44]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/CanvasButton/CanvasButton-10.asset [hunk 3]
```diff
@@ -42,7 +42,7 @@ MonoBehaviour:
           UI or even assign different toggle events. Feel free to change the toggle
           events that occur when the button is toggled.\n\nAfter you reconfigure
           the button, give the scene a try.\n\nWe encourage you to refer to the <a
-          href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-uxcomponents/packages/uxcomponents/button\">Button</a>
+          href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-uxcomponents/packages/uxcomponents/button\">Button</a>
```
Comment (aprilspeight) [inline Assets/Tutorials/CanvasButton/CanvasButton-10.asset:45]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/CanvasButton/CanvasButton-LastPage.asset [hunk 4]
```diff
@@ -40,8 +40,8 @@ MonoBehaviour:
         m_Untranslated: "In this tutorial, you learned how to configure a Canvas-based
           button. As you create your own apps and experiences, consider leveraging
           MRTK Canvas button prefabs or create your own Canvas-based UI from scratch!
-          \n\nBe sure to review our <a href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-uxcomponents/packages/uxcomponents/button\">Button</a>
-          and <a href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-uxcore/packages/uxcore/canvas-ui\">Volumetric
+          \n\nBe sure to review our <a href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-uxcomponents/packages/uxcomponents/button\">Button</a>
```
Comment (aprilspeight) [inline Assets/Tutorials/CanvasButton/CanvasButton-LastPage.asset:43]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/CanvasButton/CanvasButton-LastPage.asset [hunk 5]
```diff
@@ -40,8 +40,8 @@ MonoBehaviour:
...
+          and <a href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-uxcore/packages/uxcore/canvas-ui\">Volumetric
```
Comment (aprilspeight) [inline Assets/Tutorials/CanvasButton/CanvasButton-LastPage.asset:44]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/CanvasButtonBar/CanvasButtonBar-6.asset [hunk 6]
```diff
@@ -69,7 +69,7 @@ MonoBehaviour:
           prefab from the <b>Project</b> panel. \n\nAfter you configure the button
           bar, give the scene a try. Spawn as many aquarium objects as you prefer
           into the scene and arrange them into your preferred format.\n\nWe encourage
-          you to refer to the <a href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-uxcomponents/packages/uxcomponents/button\">Button</a>
+          you to refer to the <a href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-uxcomponents/packages/uxcomponents/button\">Button</a>
```
Comment (aprilspeight) [inline Assets/Tutorials/CanvasButtonBar/CanvasButtonBar-6.asset:72]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/CanvasButtonBar/CanvasButtonBar-LastPage.asset [hunk 7]
```diff
@@ -59,8 +59,8 @@ MonoBehaviour:
         m_Untranslated: "In this tutorial, you learned how to configure a Canvas-based
           button bar. As you create your own apps and experiences, consider leveraging
           MRTK Canvas button prefabs or create your own Canvas-based UI from scratch!
-          \n\nBe sure to review our <a href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-uxcomponents/packages/uxcomponents/button\">Button</a>
-          and <a href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-uxcore/packages/uxcore/canvas-ui\">Volumetric
+          \n\nBe sure to review our <a href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-uxcomponents/packages/uxcomponents/button\">Button</a>
```
Comment (aprilspeight) [inline Assets/Tutorials/CanvasButtonBar/CanvasButtonBar-LastPage.asset:62]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/CanvasButtonBar/CanvasButtonBar-LastPage.asset [hunk 8]
```diff
@@ -59,8 +59,8 @@ MonoBehaviour:
...
+          and <a href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-uxcore/packages/uxcore/canvas-ui\">Volumetric
```
Comment (aprilspeight) [inline Assets/Tutorials/CanvasButtonBar/CanvasButtonBar-LastPage.asset:63]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/MRTKInputSimulation/MRTKInputSimulation-LastPage.asset [hunk 9]
```diff
@@ -42,7 +42,7 @@ MonoBehaviour:
 
 
           View
-          our <a href="https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-input/packages/input/input-simulation">Input
+          our <a href="https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-input/packages/input/input-simulation">Input
```
Comment (aprilspeight) [inline Assets/Tutorials/MRTKInputSimulation/MRTKInputSimulation-LastPage.asset:45]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/PreviewProject/PreviewProject-3.asset [hunk 10]
```diff
@@ -47,7 +47,7 @@ MonoBehaviour:
           Wi-Fi connection.
 
 
-          Follow our <a href="https://docs.microsoft.com/windows/mixed-reality/develop/unity/preview-and-debug-your-app?tabs=openxr#set-up-holographic-remoting">Set
+          Follow our <a href="https://learn.microsoft.com/en-us/windows/mixed-reality/develop/unity/preview-and-debug-your-app?tabs=openxr#set-up-holographic-remoting">Set
```
Comment (aprilspeight) [inline Assets/Tutorials/PreviewProject/PreviewProject-3.asset:50]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/PreviewProject/PreviewProject-4.asset [hunk 11]
```diff
@@ -41,7 +41,7 @@ MonoBehaviour:
           users. Please skip this page by using the Next button if desired.</b>\n\nMany
           popular PC VR platforms now support OpenXR, including Windows Mixed Reality,
           SteamVR, and Oculus Rift on PC. Visit the following links for each platform
-          for setup instructions.\n\n\u2022 <a href=\"https://docs.microsoft.com/windows/mixed-reality/develop/native/openxr-getting-started\">Windows
+          for setup instructions.\n\n\u2022 <a href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/develop/native/openxr-getting-started\">Windows
```
Comment (aprilspeight) [inline Assets/Tutorials/PreviewProject/PreviewProject-4.asset:44]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/BoundsControl/BoundsControl-8.asset [hunk 12]
```diff
@@ -49,7 +49,7 @@ MonoBehaviour:
           <b>Bounds Control</b> handles to only <b>Rotation</b>. You\u2019ll have
           a chance to scale the objects to create a more customized look and feel
           in the <i>Canvas Button Bar</i> tutorial.\n\nWe encourage you to refer
-          to the <a href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-spatialmanipulation/packages/spatialmanipulation/bounds-control\">Bounds
+          to the <a href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-spatialmanipulation/packages/spatialmanipulation/bounds-control\">Bounds
```
Comment (aprilspeight) [inline Assets/Tutorials/BoundsControl/BoundsControl-8.asset:52]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/GettingStarted/GettingStarted-4.asset [hunk 13]
```diff
@@ -43,7 +43,7 @@ MonoBehaviour:
           without the need to manually configure the project for OpenXR.\n\nWithin
           our MRTK3 documentation, we provide detailed instructions on how to set
           up a Unity project and consume MRTK3 packages. To learn more, review <a
-          href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-overview/setup\">Initial
+          href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-overview/getting-started/setting-up/setup-new-project\">Initial
```
Comment (aprilspeight) [inline Assets/Tutorials/GettingStarted/GettingStarted-4.asset:46]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/ObjectManipulator/ObjectManipulator-9.asset [hunk 14]
```diff
@@ -44,7 +44,7 @@ MonoBehaviour:
           to your liking. \n\nYou\u2019re welcome to add additional objects located
           within <b>Assets > Models > Environment Prefabs</b> or import your own
           models. If you\u2019re adding your own models, be sure that your model
-          has a collider component!\n\nWe encourage you to refer to the<a href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-spatialmanipulation/packages/spatialmanipulation/object-manipulator\">Object
+          has a collider component!\n\nWe encourage you to refer to the<a href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-spatialmanipulation/packages/spatialmanipulation/object-manipulator\">Object
```
Comment (aprilspeight) [inline Assets/Tutorials/ObjectManipulator/ObjectManipulator-9.asset:47]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/ObjectManipulator/ObjectManipulator-LastPage.asset [hunk 15]
```diff
@@ -40,7 +40,7 @@ MonoBehaviour:
         m_Untranslated: "In this tutorial, you learned how to move, scale, and rotate
           an object using the <b>ObjectManipulator</b> script. We introduced some
           of the script properties which can be further configured to suit your interaction
-          needs. \n\nBe sure to review our <a href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-spatialmanipulation/packages/spatialmanipulation/object-manipulator\">Object
+          needs. \n\nBe sure to review our <a href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-spatialmanipulation/packages/spatialmanipulation/object-manipulator\">Object
```
Comment (aprilspeight) [inline Assets/Tutorials/ObjectManipulator/ObjectManipulator-LastPage.asset:43]: Please remove the locale code 'en-us' in the URL.

File: Assets/Tutorials/PreviewProject/PreviewProject-2.asset [hunk 16]
```diff
@@ -41,7 +41,7 @@ MonoBehaviour:
           any XR device attached. You can control the user\u2019s head, hands, and
           hand gestures with traditional WASD controls. To learn more, review the
           <i>How to Use MRTK Input Simulation</i> module.\n\nIn addition, we provide
-          an <a href=\"https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-input/packages/input/input-simulation-quick-ref\">Input
+          an <a href=\"https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-input/packages/input/input-simulation-quick-ref\">Input
```
Comment (aprilspeight) [inline Assets/Tutorials/PreviewProject/PreviewProject-2.asset:44]: Please remove the locale code 'en-us' in the URL.

File: Packages/manifest.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (cheles) [reply Packages/manifest.json]: tested with MRTK3-prev17 and build/play works with no issues

**Pull request merged**